src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -44,6 +44,11 @@ public sealed partial class ConfidentialClientApplication
         {
             GuardMobileFrameworks();
 
+            if (!(string.IsNullOrEmpty(configuration.AzureRegion) || configuration.CustomInstanceDiscoveryMetadata == null))
```

Comment:
Isn't there an overload to custom instance metadata with URI?

Reply:
Yeah.. included it.